source/agora/common/crypto/Schnorr.d

```diff
@@ -231,30 +242,53 @@ public bool verify (T) (const ref Point X, const ref Signature sig, auto ref T d
     nothrow @nogc @trusted
 {
     Sig s = Sig.fromBlob(sig);
+    if (!s.s.isValid()) {
```

Comment:
Ok, just remove the braces here, please. For single statements we don't use them, to avoid clutter.

Reply:
ok